### ISSUE TITLE
Is UITestingBundle Not Supported? 

### ISSUE THREAD

**Issue by azamsharp** [Tyler-Keith-Thompson/CucumberSwift#12]
The docs are very unclear as how to setup this package to be used in apps.

**Comment by Tyler-Keith-Thompson** [comment]
Thanks for the feedback. I don't think I updated the docs when I added a new requirement to the StepImplementation protocol, I'll take some time to fix that.

To answer your question this absolutely supports any testing bundle (including UITestingBundles). 

If you find (like I did) that XCUITest is slow and hard to debug you might look at this:  https://github.com/Tyler-Keith-Thompson/CucumberSwift/wiki/CucumberSwift---UIUTest:-A-winning-combination
